backend/app/params/thread_app_params.go

```diff
@@ -0,0 +1,17 @@
+package params
+
+type CreateThreadAppLayerParam struct {
+	Title       string
+	Contributer string
+}
+
+type EditThreadAppLayerParam struct {
+	ThreadKey   string
+	Title       string
+	Contributer string
+}
+
+type DeleteThreadAppLayerParam struct {
+	ThreadKey   string
+	Contributer string
```

Comment:
🚫 **[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>`Contributer` is a misspelling of `Contributor` (misspell)